agent/cmd/determined-agent/run.go

```diff
@@ -86,6 +86,10 @@ func newRunCmd() *cobra.Command {
 	cmd.Flags().StringVar(&opts.Label, "label", "",
 		"Label attached to the agent for scheduling constraints")
 
+	// ResourcePool flags.
+	cmd.Flags().StringVar(&opts.ResourcePool, "resource-pool", "default",
```

Comment:
question: are we configuring this when starting dynamic agents / checking this when the agent connects to the master or is that going to be in future work?

Reply:
we would be using this option in dynamic agents but it would be in the future work.